Install/Chroot

```diff
@@ -191,29 +191,40 @@ Edit {{{/etc/fstab}}} file according to your disk setup in step №4:
 # nano /etc/fstab
 }}}
 
-* Compile a kernel (optional)
+* Update Sorcery and Scribe
 
-If you prefer to keep your system minimal, you might want to compile your own kernel with only needed hardware support and/or to include some advanced settings.
-You can do that just inside the chroot.
+** run {{{sorcery update}}} to update the sorcery scripts
+** run {{{scribe update && cast smgl-archspecs}}} to update the spell-lists and tell [[Sorcery]] the latest list for supported CPU architectures
+** configure Sorcery (run {{{sorcery}}}, then go to **Options**)
 
-There are 2 ways:
-# includes getting any desired image from http://www.kernel.org, extracting it to {{{/usr/src}}} (e.g.) and doing that by hand (via {{{make menuconfig}}}, {{{make}}}, {{{make modules}}}, copying bzImage, etc.)
-# includes typing:
+* Update Essential [[Spell|spells]] For Compiling
 
 {{{
+# sorcery hold g++
+# cast -c mpfr gmp
+# cast -c binutils gcc
+# sorcery unhold g++
+# cast -c g++ gettext glibc
+}}}
+
+* Compile a Kernel
+
+Because we are installing from chroot, we will need to compile our own kernel. While there are a number of ways that we might be able to do this, below we simply provide the following:
+
+{{{
+# cast -c elfutils
 # cast -r linux
 }}}
 
 and following instructions in the menus.
 
 * Install a bootloader
 
-Chroot images come with 2 bootloaders: LILO and GRUB2. Feel free to use any.
-For example there's ready {{{/etc/lilo.conf}}}, where you only need to check for {{{root=}}} and {{{boot=}}} values and install it to your HDD:
+Now that we have our Linux kernel created, we need to install a bootloader. [[Spell|Spells]] are available for LILO and GRUB(2). For the purpose of this guide, we will be using LILO. While we will need to install the bootloaded, an example has already been provided at {{{/etc/lilo.conf}}}.

```

Comment:
the bootloader

Reply:
I missed this when I made my earlier corrections.
This has now been updated locally and should be taken care of through the next pull request.